docs/reference/two_step_validation.rst

```diff
@@ -35,5 +35,38 @@ Edit the configuration file:
         google_authenticator:
             enabled: true
             server:  yourserver.com
+            ip_white_list:
+                - 127.0.0.1
+            forced_for_role:
+                - ROLE_ADMIN
+
+Also, if you want to use ``ip_white_list`` and ``forced_for_role`` configuration for automatically setting secret
```

Comment:
- "configuration nodes"
- "the secret" ? (should be singular, it's a secret, right?)